### ISSUE TITLE
tf.contrib.layers.bias_add 

### ISSUE THREAD

**Issue by aliolyaei** [tensorflow/addons#2736]
**Describe the feature and the current behavior/state.**

**Relevant information**
- Are you willing to contribute it (yes/no): no
*If you wish to contribute, then read the requirements for new contributions in [`CONTRIBUTING.md`](https://github.com/tensorflow/addons/blob/master/CONTRIBUTING.md#requirements-for-new-contributions-to-the-repository)*
- Are you willing to maintain it going forward? (yes/no): no
- Is there a relevant academic paper? (if so, where): -
- Does the relavent academic paper exceed 50 citations? (yes/no): -
- Is there already an implementation in another framework? (if so, where): -
- Was it part of tf.contrib? (if so, where): tf.contrib.layers.bias_add

**Which API type would this fall under (layer, metric, optimizer, etc.)**
layer

**Who will benefit with this feature?**
all users that need use tf.contrib.layers.bias_add

**Any other info.**
I was maintaining a tf project that was written in TF1. I use tf.compat.v1 in many part of it but in some places, that project rely on tf.contrib and uses tf.contrib.layers.bias_add function. I couldn't find any suitable replace function for that, so I decide to tell you to add tf.contrib.layers.bias_add in addons or where you want.
Sorry for my bad English and thank you for your attention.

**Comment by ExpectoWu** [comment]
Did you solve this problem? I have met it too but I can find any solutions.

**Comment by mattam301** [comment]
I met this too but have no idea what tf.contrib.layers.bias_add has became :<

**Comment by seanpmorgan** [comment]
TensorFlow Addons is transitioning to a minimal maintenance and release mode. New features will not be added to this repository. For more information, please see our public messaging on this decision:
[TensorFlow Addons Wind Down](https://github.com/tensorflow/addons/issues/2807)

Please consider sending feature requests / contributions to other repositories in the TF community with a similar charters to TFA:
[Keras](https://github.com/keras-team/keras)
[Keras-CV](https://github.com/keras-team/keras-cv)
[Keras-NLP](https://github.com/keras-team/keras-nlp)
